homeassistant/components/conversation/http.py

```diff
@@ -188,6 +188,7 @@ async def websocket_hass_agent_debug(
                 conversation_id=None,
                 device_id=msg.get("device_id"),
                 language=msg.get("language", hass.config.language),
+                agent_id=None,
```

Comment:
Shouldn't this be `agent.agent_id`?

Reply:
We don't have an `agent_id` field. And anyways, it must be the default agent here.

Reply:
We have the agent ID as we have the agent object right there